tests/pyret/tests/test-statistics.arr

```diff
@@ -89,3 +113,12 @@ check "linear regression":
 
 end
 
+check "multiple regression":
+  x-s-s = [list: {4}, {4.5}, {5}, {5.5}, {6}, {6.5}, {7}]
```

Comment:
This honestly feels useless, both as a test and as an API: a whole bunch of 1-tuples? really?  And it doesn't test any of the correlations or matrix math you're doing, so it doesn't actually test the interesting part of this functionality.  The similar-ish test that I _would_ be interested in is seeing that `multiple-regression(x-s-s, y-s)({8}) is-roughly linear-regression(x-s, y-s)(8)`, showing that in the single-variable case, the multiple-regression and the linear-regression models coincide.

Reply:
I was hoping for our curriculum mavens to give me some of their data to use. Otherwise, I'll have to cook up a case and make really sure that the independent variables are really independent, or the multiple regression won't work (I probably need to throw an exception in such cases. I think this happens when the dividing determinant is roughly or exactly zero).

Reply:
Try this:

```
mr-sheet = load-spreadsheet("17t2LtdsIgKN-Iz2CUWjaQyDqPyXeObwGJsl9k7vR4SU")
mr-table = load-table: id, biking, smoking, heart-disease
  source: mr-sheet.sheet-by-name("data", true)
end
```

It's taken right from [this example page](https://www.scribbr.com/statistics/multiple-linear-regression/), which lets us compare to the output from R.

Reply:
I've added a test for 2 independent variables. (Nothing natural or realistic -- just arbitrary numbers, tested it in Lua and made sure it goes through with the same result in Pyret too.)

Also, added a test confirming that mul-reg on 1 var produces the same result as lin-reg on the same var.